README.md

```diff
@@ -115,64 +122,37 @@ _See documentation [here](#tensorflow-and-keras)._
 
 </details>
 
-> **3. Run the training like you are used to and start Aim UI**
-```shell
-$ aim up
-```
+**3. Run the training as usual and start Aim UI**
+
+_Prerequisite: In order to start Aim UI you need to have Docker installed._
 
-## Contents
-
-- [Aim](#aim)
-  - [Contents](#contents)
-  - [Getting Started In Three Steps](#getting-started-in-three-steps)
-  - [Installation](#installation)
-  - [Concepts](#concepts)
-  - [Where is the Data Stored](#where-is-the-data-stored)
-  - [Python Library](#python-library)
-    - [aim.track()](#track)
-    - [aim.set_params()](#set_params)
-    - [aim.Session()](#session)
-  - [Automatic Tracking](#automatic-tracking)
-    - [TensorFlow and Keras](#tensorflow-and-keras)
-    - [PyTorch Lightning](#pytorch-lightning)
-  - [Searching Experiments](#searching-experiments)
-    - [Search Examples](#search-examples)
-  - [Command Line Interface](#command-line-interface)
-    - [init](#init)
-    - [version](#version)
-    - [experiment](#experiment)
-    - [up](#up)
-    - [down](#down)
-    - [upgrade](#upgrade)
-    - [pull](#pull)
-  - [TensorBoard Experiments](#tensorboard-experiments)
-  - [Anonymized Telemetry](#anonymized-telemetry)
-  - [Contributor Guide](https://github.com/aimhubio/aim/wiki/Contributing)
-
-## Installation
-To install Aim, you need to have python3 and pip3 installed in your environment
-1. Install Aim python package
-```shell
-$ pip install aim
-```
-In order to start Aim UI you need to have Docker installed.
 ```shell
 $ aim up
 ```
+Jump to [[Overview](#overview)] [[Specifications](#specifications)] [[Use Cases](#use-cases)]
+
+# Overview
+
+Placeholder for the awesome description about the awesome tool. 
```

Comment:
Overview text suggestion:

Use Aim to compare 1000s of training runs at once.

When you use Aim, you create a `Session` object. `Session` object handles the tracking of metrics and parameters and stores the run in .aim repository as a directory.

You can use multiple sessions in one training script to store multiple runs at once. When not initialized explicitly, Aim creates a default session.

You can use experiments to group related runs together. An experiment named `default` is created otherwise.

Reply:
@gorarakelyan can we say that User's experience with Aim consists of 3 components: Python library, CLI and the other one (forgot the name).

Reply:
Yea, good call, but I guess there are just two components - python library(sdk) and cli

Reply:
We also provide a user interface(AimUI), but not sure if we need to document it.